components/create-state.js

```diff
@@ -0,0 +1,28 @@
+/**
+ *  file: create-state.js
+ */
+
+var Lm = require('./Lm');
```

Comment:
Yes, I considered that, but it will be for a separate check-in.  This is already the 4th review on what is essentially 7 lines of code.